### DISCUSSION TITLE
Permitir ou Bloquear o envio em caso de palavra chave

### DISCUSSION THREAD

**Discussion by GabrielRF** [GabrielRF/Rss2Telegram#2] [Geral]
Sugestão por @Adiltons

* Permitir o envio somente em casos em que uma palavra chave esteja presente?

* Bloquear o envio caso alguma palavra esteja presente? 

O que acham? As duas opções poderiam ser feitas. Seriam úteis? Qual seria a melhor forma de criar as listas? Por secrets mesmo?

**Comment by gustavosaez** [comment]
@Adiltons qual a frequência de palavras que deseja bloquear?
Você já tem uma lista?
- Não me parece algo muito efetivo, principalmente se for via SECRETS que precisa ter tudo salvo "externamente" porque ao editar vem tudo em branco.

**Reply by GabrielRF** [reply]
O secrets é só uma possibilidade. Poderia ser usado um arquivo no repositório, por exemplo.
Talvez o importante de estar no secrets seja só o token do bot mesmo.

**Reply by Adiltons** [reply]
Então, no meu caso seriam palavras com temas específicos. Exemplo: Gosto de acompanhar notícias de futebol no meu canal, mas tem alguns feeds de notícias que adicionei via SECRETS que trazem notícias de vôlei, fórmula 1, então a ideia seria criar uma lista de palavras chaves para bloquear, ou de palavras chaves permitidas para o envio.

**Comment by GabrielRF** [comment]
[Resolvido](https://github.com/GabrielRF/Rss2Telegram/discussions/5)
